gef-ui/server/controllers/confirm-cover-start-date/index.js

```diff
@@ -15,8 +15,13 @@ const setError = (field, message) => validationErrorHandler({
   errMsg: message,
 });
 
-const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverStartOnSubmission }) => {
+const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverStartOnSubmission }, dealId, _id) => {
   try {
+    const applicationUpdate = {
+      editorId: _id,
```

Comment:
Could we rename the _id param to `editorId` - would save doing this and make the params clearer 😄

Reply:
I think @ttbarnes  meant `{ _id: editorId }` unless I'm mistaken?